libs/messenger-bundle/src/Resources/config/messenger.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
proc to neni jako yaml?

Reply:
Tam vidim ze byla motivace, aby to nebylo zavisle na symfony/yaml.

Tohle neni vec, co bude pouzivat miliarda lidi, ale bude to pouzity v nasich appkach, ktery stejne yaml pouzivaj/vyzadujou.

Tahle prehnana snaha inovovat akorat vede k bordelu a neprehlednosti.
Mimochodem sice mozna to interne pisou v php, ale v dokumentaci stejne tlaci defaultne yaml.

Reply:
Prehozeny na YAML v #169